build.gradle.kts

```diff
@@ -101,7 +101,7 @@ tasks {
 val ktlint: Configuration by configurations.creating
 
 dependencies {
-  ktlint("com.pinterest.ktlint:ktlint:0.49.0-SNAPSHOT") {
+  ktlint("com.pinterest:ktlint:0.49.0-SNAPSHOT") {
```

Comment:
Still the `com.pinterest` group, fixed in https://github.com/pinterest/ktlint/pull/1929.